card/mod/virtual/lib/card/virtual.rb

```diff
@@ -0,0 +1,77 @@
+# -*- encoding : utf-8 -*-
+
```

Comment:
I had a moment where I thought maybe this name should be CardVirtualCache or CardCacheVirtual or some variant with "content" in it.

My main concern was that users who didn't understand virtual cards would assume that to be virtual, you had to be in this table.

I'm not falling in love with those, so I think we can go with this, but we need some docs here explaining what this is (and maybe pointing to some not-yet-existent documentation of virtual cards).